network-api/networkapi/wagtailpages/factory.py

```diff
@@ -199,6 +201,22 @@ class Meta:
         model = networkapi.wagtailpages.models.OpportunityPage
 
 
+class BlogPageFactory(PageFactory):
+
+    class Meta:
+        model = networkapi.wagtailpages.models.BlogPage
+        exclude = (
+            'title_text',
+        )
+
+    title = LazyAttribute(lambda o: o.title_text.rstrip('.'))
+    author = Faker('name')
+    body = Faker('streamfield', fields=blog_body_streamfield_fields)
+
+    # Lazy Values
```

Comment:
Copied this comment `# Lazy Values` from an existing section in this file. Not sure why it's called "lazy values" though

Reply:
The "lazy" refers to the fact that you're not assigning data, you're assigning a generator that will only generate real data when something tries to use it.

The code for that can be seen [here](https://github.com/mozilla/foundation.mozilla.org/blob/f7a67b3cd4db58f8be59aae4aea3b7e21bc19506/network-api/networkapi/wagtailpages/factory.py#L23-L25)